composer.json

```
@@ -28,7 +28,7 @@
     },
     "require-dev": {
         "composer/semver": "^3.0",
-        "phpunit/phpunit": "^9.6.22",
+        "phpunit/phpunit": "^10.5 || ^11.5",
```

```suggestion
        "phpunit/phpunit": "^10.5.40 || ^11.5",
```

Let's make sure, the low deps run is run on a fairly recent version of PHPUnit.